terraform/development/per_review_app/main.tf

```diff
@@ -14,7 +14,6 @@ terraform {
     encrypt        = true
     key            = "core-development-per-review-app.tfstate"
     region         = "eu-west-2"
-    role_arn       = "arn:aws:iam::815624722760:role/developer"

```

Comment:
The assumption here is that terraform will be run from a role that natively has access to the necessary items for the state (rather than defining a new role, in part because terraform is annoying about role chaining).

Reply:
N.B. As developers, we can do this by running terraform while in the developer role in the meta account.